pages/api/suggestion-image.ts

```diff
@@ -0,0 +1,62 @@
+// Next.js API route support: https://nextjs.org/docs/api-routes/introduction
+import axios from 'axios'
+import type { NextApiRequest, NextApiResponse } from 'next'
+
+export default async function handler(req: NextApiRequest, res: NextApiResponse) {
+  const { keyword } = req.body
+  const PAPAGO_API_URL = process.env.PAPAGO_TRANSLATION_API_URL
+  const PAPAGO_CLIENT_ID = process.env.PAPAGO_CLIENT_ID
+  const PAPAGO_CLIENT_SECRET = process.env.PAPAGO_CLIENT_SECRET
+  const KAKAO_REST_API_KEY = process.env.KAKAO_REST_API_KEY
+
+  const headers = {
+    'Content-Type': 'application/x-www-form-urlencoded',
+    'X-Naver-Client-Id': PAPAGO_CLIENT_ID,
+    'X-Naver-Client-Secret': PAPAGO_CLIENT_SECRET,
+  }
+  let text
+
+  try {
+    const response = await axios.post(
+      PAPAGO_API_URL as string,
+      {
+        source: 'ko',
+        target: 'en',
+        text: keyword,
+      },
+      { headers }
+    )
+
+    text = response.data.message.result.translatedText
+  } catch (error) {
+    console.log(error)
+
+    return res.status(500).json({ message: '키워드 영변환에 실패했습니다.' })
+  }
+
+  try {
+    const response = await axios.post(
+      'https://api.kakaobrain.com/v1/inference/karlo/t2i',
+      {
+        prompt: {
+          text: text + 'by Claude Monet',
+          batch_size: 8,
+        },
+      },
+      {
+        headers: {
+          Authorization: `KakaoAK ${KAKAO_REST_API_KEY}`,
+          'Content-Type': 'application/json',
+        },
+      }
+    )
+
+    return res.status(200).json({ ...response.data })
+  } catch (error) {
+    console.log(error)
+
+    return res.status(500).json({
+      message: '이미지 추천에 실패했습니다. 다른 키워드를 입력해 주세요.',
+    })
```

Comment:
사소한거지만 혹시 해당 api에서 에러가 발생할 때 500이 뜨는걸까요?
관련 에러를 클라이언트에서도 명확히 확인할 수 있게
```typescript
const statusCode = response.status;
const statusMessage = response.statusText;
```
와 같이 지정한 다음 응답에 태우는 것도 좋을거같아요~!

Reply:
- 에러 상황 시 해당 에러에 대한걸 클라이언트에 보여주는걸 말씀하시는거죠??
- 그럼 send로 변경하는것도 괜찮을까요? 아래처럼요!!
```suggestion
    return res.send({...error})
```

Reply:
- 바쁘신거 같아서 일단 send로 에러 전체를 보내는 방식으로 올려놓겠습니다 더 좋은방법 있으면 수정 후에 알려주시면 감사합니다 !!🙇‍♂️

Reply:
- send 객체로 보내는 방법도 좋은거 같아요! 다만 에러 구분관련은 해당 에러 객체 내부에 정보(키워드, 이미지)가 있을 것 같아서 따로 구분하지 않아도 괜찮아보여요!
- 만약 키워드 번역과 이미지 생성 구분을 명시적으로 하고싶다면, statusCode만 받아와서 지정해주고 message는 기존 작업하신대로 놔두도록 간단하게 구현해도 좋아보입니당 👍

Reply:
- 확인했습니다 의견 감사합니다 ㅎㅎ send로 보내는 방식으로 적용해놓겠습니다!!🙆‍♂️🙆‍♂️